docs/book/src/topics/powervs/creating-a-cluster.md

```diff
@@ -131,31 +135,11 @@ following the steps below.
     ibm-powervs-1-md-0-4dc5c             Ready    <none>   41h   v1.26.2
     ibm-powervs-1-md-0-dbxb7             Ready    <none>   20h   v1.26.2
 
-### Deploy a PowerVS cluster with user provided resources
-
-  ```
-  IBMPOWERVS_SSHKEY_NAME="my-pub-key" \
-  IBMPOWERVS_VIP="192.168.167.6" \
-  IBMPOWERVS_VIP_EXTERNAL="163.68.65.6" \
-  IBMPOWERVS_VIP_CIDR="29" \
-  IBMPOWERVS_IMAGE_NAME="capibm-powervs-centos-streams8-1-26-2" \
-  IBMPOWERVS_SERVICE_INSTANCE_ID="3229a94c-af54-4212-bf60-6202b6fd0a07" \
-  IBMPOWERVS_NETWORK_NAME="capi-test" \
-  clusterctl generate cluster ibm-powervs-1 --kubernetes-version v1.26.2 \
-  --target-namespace default \
-  --control-plane-machine-count=3 \
-  --worker-machine-count=1 \
-  --flavor=powervs | kubectl apply -f -
-  ```
-
 ### Deploy a PowerVS cluster with infrastructure creation
 
 #### Prerequisites: 
-- Set `EXP_CLUSTER_RESOURCE_SET` to true as the cluster will be deployed with external cloud provider which will create the resources to run the cloud controller manager.
-- Set the `provider-id-fmt` [flag](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/5e7f80878f2252c6ab13c16102de90c784a2624d/main.go#L168-L173) to `v2` via `PROVIDER_ID_FORMAT` environment variable.
+- Set `EXP_CLUSTER_RESOURCE_SET` to `true` as the cluster will be deployed with external cloud provider which will create the resources to run the cloud controller manager.
```

Comment:
Since this is added commonly at L42, can it be omitted in this and below sections to avoid repetition?

Reply:
Fixed.